src/wallet/coincontrol.cpp

```diff
@@ -10,6 +10,7 @@ void CCoinControl::SetNull()
 {
     destChange = CNoDestination();
     m_change_type.reset();
+    m_add_inputs = true;
```

Comment:
Haven't tested and looks like anyone asked, but can't we use `fAllowOtherInputs` instead?

Reply:
That won't work. I think `fAllowOtherInputs` was designed with RBF in mind. It _permits_ adding coins, as long as all the currently selected coins are included. But this PR tries to _prevent_ adding coins.